activerecord/test/cases/arel/visitors/to_sql_test.rb

```diff
@@ -5,1035 +5,1037 @@
 
 module Arel
   module Visitors
-    describe "the to_sql visitor" do
-      before do
-        @conn = FakeRecord::Base.new
-        @visitor = ToSql.new @conn.lease_connection
-        @table = Table.new(:users)
-        @attr = @table[:id]
-      end
-
-      def compile(node)
-        @visitor.accept(node, Collectors::SQLString.new).value
-      end
-
-      it "works with BindParams" do
-        node = Nodes::BindParam.new(1)
-        sql = compile node
-        _(sql).must_be_like "?"
-      end
+    class ToSQLTest < Arel::Spec
```

Comment:
>There are 4 Arel tests in activerecord/test/cases/arel/visitors/to_sql_test.rb that do not have any assertions. 

There are a lot of changes to this file which makes it really hard to see which tests were missing assertions and which ones are just shuffled around. 
Should we consider splitting changes into separate commits or even PRs so we first add missing assertions and only then change the test file itself

Reply:
If you view the diff with whitespace hidden then the changes are just 6 lines https://github.com/rails/rails/pull/53976/files?diff=split&w=1#diff-794a3622add7d37e41208b56e0efde3bf94cd98f8a235fef6522a1bd1576e9fd